electron/main.ts

```diff
@@ -551,6 +551,15 @@ const readAppVersion = async (event) => {
     });
 };
 
+const getUserData = async () => {
```

Comment:
Why not use `fetch-username` which already exists?

I prefer your function so maybe you could probably update that function (which still follows the old-style synchronous stuff) to match your new function?

Reply:
I think it might still suggest merging them (or removing the old one?)